src/components/Election/ElectionResults.tsx

```diff
@@ -0,0 +1,117 @@
+import { Tdata } from "../../types";
+
+import { FormattedMessage } from "react-intl";
+
+export function ElectionResults({ data }: { data: Tdata }) {
+  const {
+    totalWorkers,
+    worksCouncilSize,
+    minorityGender,
+    workplaceGenderQuota,
+    totalCandidates,
+    notEnoughSeats,
+    moreVotesThanWorkers,
+    suggestMoreSeats,
+    suggestedSeats,
+    totalVotes,
+  } = data;
+
+  const minorityGenderHasMembers = data[`num${minorityGender}`] > 0;
+
+  const numMinorityWorkers = workplaceGenderQuota[minorityGender];
+  const isQuotaDisabled =
+    worksCouncilSize > 1 && minorityGenderHasMembers && numMinorityWorkers;
+
+  return (
+    <div>
+      <h2>Election Stats</h2>
+      <div className="input-control">
+        <label htmlFor="totalWorkers">
+          <FormattedMessage
+            id="label.workerCount"
+            defaultMessage={"Worker Count"}
+          />
+        </label>
+        <span className="cell" id="totalWorkers">
+          {totalWorkers}
+        </span>
+      </div>
+      <div className="input-control">
+        <label htmlFor="numSeats">
+          <FormattedMessage
+            id="label.worksCouncilSeatCount"
+            defaultMessage={"Works Council Seats"}
+          />
+        </label>
+        <span className="cell" id="numSeats">
+          {worksCouncilSize}
+        </span>
+      </div>
+      {/* TODO: how does minority gender work with single member works councils? */}
```

Comment:
[Line 23 ](https://github.com/techworkersco/works-council-list-helper/pull/15/files#diff-b97cb079c4b59c39215dfae2a55e44fc2c0eb66f8b6488812b39c009e741f290R23)already handles this, because it's only applicable when 3 or more members are in a WoCo. 

[BetrVG § 15(2)](https://www.gesetze-im-internet.de/englisch_betrvg/englisch_betrvg.html#p0117) is the basic summary of gender quota, confirming it's for 3 member and up sized works councils